hw/ip/sram_ctrl/dv/sram_ctrl_base_sim_cfg.hjson

```diff
@@ -111,6 +110,18 @@
       name: "{variant}_executable"
       uvm_test_seq: sram_ctrl_executable_vseq
     }
+    // Below 2 tests are same as the tests in dvsim/tests/mem_tests.hjson
+    // Replicate them here in order to enable scb
+    {
+      name: "{variant}_mem_walk"
+      uvm_test_seq: "{name}_common_vseq"
+      run_opts: ["+csr_mem_walk"]
+    }
+    {
+      name: "{variant}_mem_partial_access"
+      uvm_test_seq: "{name}_common_vseq"
+      run_opts: ["+run_mem_partial_access"]
+    }
```

Comment:
I believe you can add the run option and still include `"{proj_root}/hw/dv/tools/dvsim/tests/mem_tests.hjson",`, like what GPIO did here: 
https://github.com/lowRISC/opentitan/blob/master/hw/ip/gpio/dv/gpio_sim_cfg.hjson#L125

Reply:
We have en_scb=0 in mem_test, I want to remove this run opt to enable the scb, but we can't override the run_opts, since we need other run options.

Reply:
Oh I see, thanks for explaning.